### DISCUSSION TITLE
ask for a refund

### DISCUSSION THREAD

**Discussion by ppbbhk** [aclap-dev/video-downloadhelper#857] [License & Purchase requests]
### What's your issue?

Your question
Hi, I bought a download helper premium last week.
I found i got the wrong version.
I want to refund my previously purchased license.

thank you.
my email is : godwithpeter@gmail.com

### The 10 first digits (only 10) of your license number or email address

1ed129f3eb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Hi,
You have been refunded. Should be available in a few business days.
The links to be sure to purchase the good one: https://github.com/aclap-dev/video-downloadhelper/wiki#purchase-premium

**Comment by ppbbhk** [comment]
thank you

jcfrog ***@***.***> 于2024年6月11日周二 03:11写道：
